### DISCUSSION TITLE
Cursor pagination, query from newest

### DISCUSSION THREAD

**Discussion by IRediTOTO** [prisma/prisma#14710] [Q&A]
Hi,i am trying to learn cursor pagination. Is there any way use cursor to get newest post base on `updatedAt` columns?
Or query from biggest `id`, I don't know how can I get the biggest `id` number too. Use `count` won't work because it passed some deleted row.

**Comment by ludralph** [comment]
Hi @IRediTOTO 👋 ,

I will recommend going through our excellent documentation on [cursor-based pagination](https://www.prisma.io/docs/concepts/components/prisma-client/pagination#cursor-based-pagination). It should give you a pretty good idea of how to work with cursor-based pagination and if you have any other question, I'll be happy to help.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
